govuk-paas/paas-rds-broker
# 185765111 tls support

The implementation of RDS broker does not use github.com/alphagov/paas-service-broker-base. For the time being I have "borrowed" and converted parts of it to make `RDS-Broker` support TLS connectivity.

How to test:
- Check if the integration tests based on this branch have been successful
- Use the artefact produced in a paas-cf deployment 
- Check that the broker works as expected on that deployment
